shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/Sanitizer.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.konan.target
+
+enum class SanitizerKind {
+    NONE,
```

Comment:
Maybe get rid of this entry in favor of nullability ?

Reply:
I had issues with that. First of all, gradle does not support calling task constructors with `null`: https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:passing_arguments_to_a_task_constructor So I had to move `sanitizer` into a `public var`. Which broke `CompileToBitcode`<->`CompilationDatabase` interaction, because the latter reads target directory of the former during configuration phase, but the target directory depends on the sanitizer value which now can change during the configuration phase. This interaction is likely flawed in and of itself but will require some refactoring to get fixed.